src/seedsigner/hardware/framebuffer.py

```diff
@@ -0,0 +1,40 @@
+from PIL import Image
+import mmap
+from seedsigner.hardware.RGBtoBGR import rgbtobgr
+
+class Framebuffer(object):
+    def __init__(self, device_no: int):
```

Comment:
Current code convention would use `device_num` as the var name. Also it's hard-coded to 0 when instantiated in `Renderer` so it should just be set as a default here and not referenced at all on instantiation.

```
def __init__(self, device_num: int = 0):
```

Reply:
There might be configurations where the framebuffer device is not `0`. For now default 0 makes sense.